app/controllers/api/v1/search_results_controller.rb

```diff
@@ -19,8 +14,26 @@ def create
         end
       end
 
+      def index
+        # TODO: Add pundit to filter search results by user
+        search_result_list = SearchResult.where(user_id: current_user.id).order(:id)
```

Comment:
Here's a cooler way of doing it 🤩 
```suggestion
        search_result_list = current_user.search_results.order(:id)
```

Reply:
@Tuubz Looks better 👍